dmutils/email/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Allow accessing send email from email. Backwards compatibility."""
+from dmutils.email.dm_mandrill import send_email
```

Comment:
I'd do the same for the notify client, to keep imports the same if it ends up being the only email client in the future